operator-common/src/main/java/io/strimzi/operator/common/process/ProcessHelper.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.process;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class ProcessHelper {
+
+    private static final Logger LOGGER = LogManager.getLogger(ProcessHelper.class);
+
+    public static ProcessResult executeSubprocess(List<String> verifyArgs) throws IOException, InterruptedException {
+        // We choose to run the reassignment as an external process because the Scala class:
```

Comment:
I guess this is not anymore only for reassignments, so we should fix the comments.